pom.xml

```diff
@@ -283,7 +283,7 @@ THE SOFTWARE.
           <providerClass>${hudson.sign.providerClass}</providerClass>
           <providerArg>${hudson.sign.providerArg}</providerArg>
           <tsa>${hudson.sign.tsa}</tsa>
-          <!-- 
+          <!--
```

Comment:
Unrelated, I have not disabled the plugin in Atom on my Windows machine. But it should not hurt